torch/nn/modules/batchnorm.py

```diff
@@ -60,7 +60,7 @@ def _check_input_dim(self, input):
     def forward(self, input):
         self._check_input_dim(input)
 
-        exponential_average_factor = 0.0
+        exponential_average_factor = 0.0 if self.momentum is None else self.momentum
```

Comment:
`expected a value of type float for argument 'momentum' but found float?` error in CI. you probably also want to check whether self.momentum is None or not.

Reply:
Fixed. Issue with the use of ternary operator. See comment below.